pkg/registry/apis/secret/register.go

```diff
@@ -695,10 +718,17 @@ func (b *SecretAPIBuilder) Mutate(ctx context.Context, a admission.Attributes, o
 	return nil
 }
 
-// TODO: use this for starting the outbox queue async process?
 func (b *SecretAPIBuilder) GetPostStartHooks() (map[string]genericapiserver.PostStartHookFunc, error) {
 	return map[string]genericapiserver.PostStartHookFunc{
-		"start-outbox-queue-workers": func(context genericapiserver.PostStartHookContext) error {
+		"start-outbox-queue-worker": func(hookCtx genericapiserver.PostStartHookContext) error {
+			go func() {
```

Comment:
IIRC this already runs in a goroutine, I think you dont need to wrap it again 🤔

Reply:
remove the goroutine